test/PowerShellEditorServices.Test/Language/SemanticTokenTest.cs

```diff
@@ -0,0 +1,190 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Management.Automation.Language;
+using System.Threading.Tasks;
+using Microsoft.PowerShell.EditorServices.Services.TextDocument;
+using Microsoft.PowerShell.EditorServices.Handlers;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+using OmniSharp.Extensions.LanguageServer.Protocol.Document.Proposals;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models.Proposals;
+using Xunit;
+
+namespace Microsoft.PowerShell.EditorServices.Test.Language
```

Comment:
The tests here look good. One extra thing I would do is include tests for the less conspicuous tokens like `(`, `)`, `{`, `}` -- we have to classify and highlight those too, unless perhaps there's some other mechanism at work there?

Reply:
Those tokens aren't mapped to SemanticTokenTypes and are handled by EditorSyntax

Reply:
Oh, I didn't realise that. That seems a bit strange, and maybe like we're doing more work than we should... We should maybe discuss at some point

Reply:
I map the tokens that can be recognized as one of these https://github.com/OmniSharp/csharp-language-server-protocol/blob/f93f973ec96d895c0c6056f5f5942c4646071ab9/src/Protocol/Models/Proposals/SemanticTokenTypes.cs